### DISCUSSION TITLE
An empty panel is displayed on Detail when Resource fields only contain Panels

### DISCUSSION THREAD

**Discussion by WillyReyno** [laravel/nova-issues#4048] [Ideas & Feature Requests]
- Laravel Version: 8.83.8
- Nova Version: 4.2.2
- PHP Version: 8.0
- Database Driver & Version: MySQL 8.0

### Description:

When Resource fields only contain Panel(s), an empty Panel appears at the top of the Resource Detail page : 

![Capture d’écran 2022-04-17 à 02 45 27](https://user-images.githubusercontent.com/5576482/163695647-54b09b5c-9815-438b-9668-f5ad38d5b6a1.png)

If another field is added within the `fields()` array, the empty panel disappears: 
![Capture d’écran 2022-04-17 à 02 46 29](https://user-images.githubusercontent.com/5576482/163695648-63be98ea-e1cc-4bfc-a45e-eee3548c7f3d.png)

### Detailed steps to reproduce the issue on a fresh Nova installation:
* Create a resouce
* Within the `fields()` method, declare one or more panels (without any field at the root of the array)

**Comment by crynobone** [comment]
I am hesitent to fix this issue because it would create more visual issues rather than fixing anything. Default panel are special panel used as primary placeholder and has custom toolbar.

![image](https://user-images.githubusercontent.com/172966/163734727-84764358-c94d-4ebc-b5ee-5296c50c50e4.png)

Just show how mess up it can be by adding Panel as first field.
